manifest.json

```diff
@@ -19,31 +19,37 @@
       "size": 48
     }]
   },
+  "options_ui": {
+    "page": "options.html",
+    "open_in_tab": true,
+    "browser_style": false,
+    "chrome_style": false
+  },
+  "applications": {
+    "gecko": {
+      "id": "git@ekill.com",
+      "strict_min_version": "42.0"
+    }
+  },
   "permissions": [
-    "activeTab"
+    "activeTab",
+    "storage",
+    "unlimitedStorage"
```

Comment:
Hey @duart38,

This slipped my attention on initial review. Is there any specific reason we need this? It seems to only relate to "Web SQL Database and application cache" according to this: https://developer.chrome.com/apps/declare_permissions

Reply:
i'm not sure, i also got a bit confused by the docs here. i read in some places that the extension or app is limited to 5 MB of local storage, hence why it's there.

Reply:
Ok, yeah I see it mentioned here in https://developer.chrome.com/apps/storage#property-local as well.

I doubt anyone will hit 5MB, but I'll just leave it as is then.